conda/recipes/cuspatial/build.sh

```diff
@@ -5,4 +5,4 @@
 unset CMAKE_ARGS
 
 # This assumes the script is executed from the root of the repo directory
-./build.sh cuspatial
+./build.sh cuspatial cuproj
```

Comment:
If cuproj is distributed as a separate wheel from cuspatial (my understanding from above), then it should also be a separate conda package. You'll need to copy the cuspatial recipe and create `conda/recipes/cuproj/*`.

Reply:
Conda recipe added, please review! Thanks!